.vib/trivy/goss/trivy.yaml

```diff
@@ -0,0 +1,17 @@
+command:
+  check-trivy-help:
+    exec: trivy --help
+    exit-status: 0
+    stdout:
+      - Commands
+  check-wire-help:
+    exec: wire --help
+    exit-status: 0
+    stderr:
+      - Subcommands
```

Comment:
I would say checking the `--help` flags does not really add enough value, given that we are already checking the binaries themselves exist.

It's true the comparison is not 1:1, but I think checking a flag and, specifically, a specific output (prone to be changed by upstream) introduces more flakiness than value. Additionally, you are already using the `trivy` binary in the `trivy-scanner` check.

I think we can remove these tests altogether or at least substitute them with ones that test the core functionalities of the asset (like the scanning of an image by trivy below). WDYT @migruiz4 @joancafom?

Reply:
I removed both the `trivy --help` and `wire --help` tests.

The `trivy --help` does not provide any additional value considering we are running a trivy scan below. 
The `wire --help` test was removed because we are already checking the binary is present in the `check-binaries` test.